GVRf/Framework/backend_oculus/src/main/java/org/gearvrf/OvrXMLParser.java

```diff
@@ -106,6 +106,10 @@
                                 settings.setUseGazeCursorController(Boolean
                                         .parseBoolean(xpp.getAttributeValue(i)));
                             } else if (attributeName
+                                    .equals("useAndroidWearTouchpad")) {
```

Comment:
You will need to update the parsers for the other backends to recognize the new setting. Clearly we need a common parser but for now they are separate.